lib/authlogic/version.rb

```diff
@@ -17,6 +17,6 @@ module Authlogic
   #
   # @api public
   def self.gem_version
-    ::Gem::Version.new("6.4.2")
+    ::Gem::Version.new("6.4.3")
```

Comment:
I'll probalby cut the version in a separate commit: 

```suggestion
```

Reply:
Removed version bump from my PR